packages/serverpod/lib/src/relic/web_server.dart

```diff
@@ -126,15 +126,24 @@ class WebServer {
       }
     }
 
-    // TODO: Fix body
-    var session = MethodCallSession(
-      server: serverpod.server,
-      uri: uri,
-      path: 'webserver',
-      body: '',
-      authenticationKey: authenticationKey,
-      httpRequest: request,
-    );
+    MethodCallSession session;
+    try {
+      // TODO: Fix body
+      session = MethodCallSession(
+        server: serverpod.server,
+        uri: uri,
+        path: 'webserver',
+        body: '',
+        authenticationKey: authenticationKey,
+        httpRequest: request,
+      );
+    } catch (e) {
+      // Triggered if the URI query parameters are malformed
+      logDebug('Failed to create method call session: $e');
```

Comment:
Chore: Ensure we log in the same way as for the api server.

As linked in the original PR this logic should mimic the same logging behavior as for endpoints: https://github.com/serverpod/serverpod/pull/2398#issuecomment-2222144353

That is, we should check the logging settings before we write anuthing to the console. And the logg should mimic the what we have there:

https://github.com/serverpod/serverpod/blob/1bad0d40a5ee28e69fa534e1f364f87b640e9fcd/packages/serverpod/lib/src/server/server.dart#L305-L314.